experimentalSetups/lego_setup_amc_advfoc/hardware/motorControl/setup-eb2-j0_2-mc_service.xml

```diff
@@ -113,15 +113,15 @@
             <group name="JOINTMAPPING">
 
                 <group name="ACTUATOR">
-                    <param name="type">             eomc_act_advfoc     </param>
+                    <param name="type">             advfoc     </param>
                     <param name="onboard">          amc2c               </param>
                     <param name="port">             ICC1:3              </param>
                 </group>
 
                 <group name="ENCODER1">
                     <param name="type">             aea3                </param>
                     <param name="port">             CONN:J5_X1          </param>
-                    <param name="position">         eomc_pos_atjoint    </param>
+                    <param name="position">         atjoint             </param>
```

Comment:
I used these files while working on the setup that uses one single motor, by mistake I included in this PR.
These changes were made by me under guidance of @marcoaccame. The functionality of the code hasn't changed, but i can manage to exclude these files from the PR. I'll also wait a response of Marco in this to be sure.

Reply:
it is the same either string

Reply:
it is ok to keep the file but pls add a comment about why you changed it it in the description of the PR:
> Changed so that the `lego_setup_amc_advfoc` can run w/ latest version of `amc2c` and `amcbldc` application